service/controller/aws/v16/resource/clusterconfigmap/current.go

```diff
@@ -0,0 +1,38 @@
+package clusterconfigmap
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/giantswarm/microerror"
+	"k8s.io/api/core/v1"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/giantswarm/cluster-operator/pkg/v16/key"
+	awskey "github.com/giantswarm/cluster-operator/service/controller/aws/v16/key"
+)
+
+func (r *StateGetter) GetCurrentState(ctx context.Context, obj interface{}) ([]*v1.ConfigMap, error) {
+	cr, err := awskey.ToCustomObject(obj)
+	if err != nil {
+		return nil, microerror.Mask(err)
+	}
+
+	clusterConfig := awskey.ClusterGuestConfig(cr)
+	name := key.ClusterConfigMapName(clusterConfig)
+
+	r.logger.LogCtx(ctx, "level", "debug", "message", fmt.Sprintf("finding cluster configMap %#q", name))
```

Comment:
```suggestion
	r.logger.LogCtx(ctx, "level", "debug", "message", fmt.Sprintf("finding cluster configMap %#q in namespace %#q", name, cr.Namespace))
```

Sorry I missed this earlier and it will be needed in a lot of places.

Since we are storing this in the cluster namespace rather than `giantswarm` we should include it in the logging.

Reply:
I don't think we can use `cr.Namespace` since it's default somehow. 
```
k get awsclusterconfigs.core.giantswarm.io -o yaml
apiVersion: v1
items:
- apiVersion: core.giantswarm.io/v1alpha1
  kind: AWSClusterConfig
  metadata:
    creationTimestamp: 2019-05-02T16:22:49Z
    finalizers:
    - operatorkit.giantswarm.io/cluster-operator
    generation: 1
    name: ho26x-aws-cluster-config
    namespace: default
    resourceVersion: "119111055"
```

I will get cluster ID as `kubeconfig` resource do already.